### PR TITLE
Fix Maybe and Either docs

### PR DESCRIPTION
**Pull request by 1ven**

### CODE REVIEW

**Comment by davidchambers** [comment]
Thanks for the pull request, @1ven.

__README.md__ is updated automatically at release time, so there's no need to update it in this pull request. Please revert the changes to that file (and `git commit --amend`).

File: README.md [hunk 1]
```diff
@@ -3311,6 +3311,7 @@ See also [`splitOn`](#splitOn).
 [Semigroupoid]:     https://github.com/fantasyland/fantasy-land/tree/v3.3.0#semigroupoid
 [Setoid]:           https://github.com/fantasyland/fantasy-land/tree/v3.3.0#setoid
```
Comment (davidchambers) [inline README.md]: This link definition can now be removed, I believe.

File: README.md [hunk 2]
```diff
@@ -3311,6 +3311,7 @@ See also [`splitOn`](#splitOn).
...
 [Traversable]:      https://github.com/fantasyland/fantasy-land/tree/v3.3.0#traversable
+[Ord]:              https://github.com/fantasyland/fantasy-land/tree/v3.3.0#ord
```
Comment (davidchambers) [inline README.md]: Please insert this in alphabetical order. :abc:

**Comment by 1ven** [comment]
@davidchambers hey, I've fixed the issues above

**Comment by davidchambers** [comment]
Thanks! The changes look good.

Would you mind squashing the commits? You could run these commands to do so:

```console
$ git rebase 82ed5a8bd6fe21e75d1a3a889c4b5ea89852ba8c^ --interactive
$ # change "pick" to "fixup" for the second and third commits
$ git push origin master --force
```

In future I suggest making commits on a feature branch, as `--force` pushing to `master` is not a habit one should develop. ;)

**Comment by 1ven** [comment]
@davidchambers thanks for suggestions !:)

I've made corresponding changes.

**Pull request merged**